common/app/views/fragments/javaScriptLaterSteps.scala.html

```diff
@@ -5,6 +5,27 @@
 
 @* Scripts that should be executed after CSS is loaded *@
 
+@if(item.isJspm) {
+    @if(play.Play.isDev()) {
+        <script src="/assets/jspm_packages/system.js"></script>
+        <!-- Due to Babel conflcting with curl, we have to manually load.
+        As per https://github.com/systemjs/systemjs/issues/378#issuecomment-83982349 -->
+        <script src="/assets/jspm_packages/babel.js"></script>
+        <script src="/assets/jspm-config.js"></script>
+        <script>
+            System.import('es6/bootstraps/app')
+                .catch(function (error) {
+                    // Throw error outside of promise
+                    setTimeout(function () {
+                        throw error;
+                    });
+                });
+        </script>
+    } else {
+        <script src="@Static("bundles/app.js")"></script>
```

Comment:
@theefer Nope they are definitely reliable for modern browsers. Just we need to couple with some conditional "mustard cutting" logic anyway so made sense to dynamically construct. But downside to this it is not fetched by the pre-parser.

This is a temporary solution, as when we start porting the entire application over to jspm we are going to load System normally via async script tag, then use conditional logic to dynamically load certain parts of application in sequence.